bin/buildBlueprints.js

```diff
@@ -111,6 +112,18 @@ if (argv.watch) {
   extensions.watch = true;
 }
 
+function addCacheBustedFile(name, m) {
```

Comment:
I find jsdoc overly verbose. Good code is self-documenting, etc.

`m` should be named `mochaInstance` here, for instance. A normal comment should be added above that explains why this is necessary:

`// This function concatenates a timestamp to a filename, and adds the new filename to the mocha instance. This is necessary because the require() call is cached by filename, and we want to load the new file.`